md/admin-application-roles-list.md

```diff
@@ -0,0 +1,52 @@
+# Role list in the Bonita Administrator Application
+
+This page explains what a user with the _Administrator_ profile in Bonita Administrator Application can see and do about roles in the organization.  
+To easily manipulate users with the same rights, it is possible to add them in the same role, like "Project Managers", for example.   
+Roles can be defined in the organization, users can then be added to roles, and roles are used to map actors or profiles or other permissions in Bonita.
```

Comment:
```suggestion
Roles can be defined in the organization, users can then be given the roles (within a group), and roles can be used to map actors or profiles or other permissions in Bonita.
```

Reply:
roles can be given even if the user belongs to no group yet. I'm keeping the "can be" though.

Reply:
yes.
What I was trying to say is that it not possible to give a role to a user without adding him in a group at the same time or changing an existing membership. but you already explaining the membership concept the bottom of the page.